docs/source/azure/usecases/securityrules.rst

```diff
@@ -0,0 +1,27 @@
+Find Open SSH Ports
+===================
+
+This policy will deny access to all security rules with Inbound SSH ports in the range [8080,8090]
+
+.. code-block:: yaml
+
+     policies:
+       - name: open-ingress
+         resource: azure.networksecuritygroup
+         filters:
+          - type: ingress
+            FromPort: 8080
+            ToPort: 8090
+         actions:
+          - type: close
+
+Available types:
+    - ingress - Inbound Security Rules
+    - egress - Outbound Security Rules
+
+Available filters:
+    - FromPort - Lower bound of port range (inclusive and can be used alone to indicate all ports at or above number)
+    - ToPort - Upper bound of port range (inclusive and can be used alone to indicate all ports at or below number)
+    - Ports - Filter on ports contained in list.  Ex: Ports [8080,8081]
+    - OnlyPorts - Filter on ports NOT contained in list. Ex: OnlyPorts [22] 
```

Comment:
I feel like OnlyPorts is a ambiguous and non-intuitive, especially since Ports is also an option. Perhaps RestrictedPorts would be better?

Reply:
Agreed. I was iffy on the naming scheme, but I tried to copy convention from vpc.py on the AWS side. I like RestrictedPorts

Reply:
Ah, you're right. I'm inclined to stick with the convention present in vpc.py for the sake of consistency w/ AWS but it took me a while to understand what OnlyPorts was doing. @kapilt thoughts?